source/topologies.txt

```diff
@@ -0,0 +1,32 @@
+============================
+Supported Cluster Topologies
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:ref:`c2c-mongosync` supports two cluster topologies: 
+
+- :ref:`Replica sets <replication>`
+
+- :ref:`Sharded clusters <sharded-cluster>`
+
+To sync a replica set to a replica set, see the :ref:`Quick Start Guide
+<c2c-quickstart>`.
+
+To sync a replica set to a sharded cluster, see :ref:`c2c-rs-to-sharded`.
+
+To sync a sharded cluster to a sharded cluster, see :ref:`c2c-sharded-clusters`.
+Sharded clusters are not required to have the same number of shards.
```

Comment:
[question/suggestion] Do you think this information should be in an admonition of some kind? I would suggest a note.

Reply:
Creating note for "``mongosync`` doesn't support sync from a sharded cluster to a replica set."

Reply:
Adding note for:

 - If you are syncing sharded clusters, they are not required to have the same 
   number of shards.

   - ``mongosync`` doesn't support sync from a sharded cluster to a replica set.

What do you think?